### DISCUSSION TITLE
NextJS

### DISCUSSION THREAD

**Discussion by stanislavyv** [commercetools/ui-kit#2679] [Q&A] [answered]
Hello, is this library compatible with Next.js's new app router?

**Comment by Rhotimee** [comment, marked as answer]
Hey @stanislavyv, the UI kit should generally be compatible with any react-based framework including Next.js and its new app router.  
Please don't hesitate to let us know if you encounter any specific issues while integrating the UI kit with Next.js (app router). Thanks!

**Reply by stanislavyv** [reply]
Thanks for the quick response! Are there any prerequisites for the setup part in order to enable SSR support?

**Reply by Rhotimee** [reply]
@stanislavyv the UI kit is not fully compatible with SSR if you want to configure theming. See [issue](https://github.com/commercetools/ui-kit/issues/2329)

**Reply by stanislavyv** [reply]
Thanks for the info 🙂
